test/local/DSL/marketSwapAction.test.ts

```diff
@@ -0,0 +1,118 @@
+import { expect } from 'chai';
+import { marketSwapAction, UniV3Twap, Token } from '../../../src';
+
+const USDC_ADDRESS = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48';
+const WETH_ADDRESS = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2';
+const DAI_ADDRESS = '0x6b175474e89094c44da98b954eedeac495271d0f';
+
+describe('marketSwapAction', function () {
+
+  it('should create correct market swap action with default values', function () {
+    const primitives = marketSwapAction({
+      type: 'marketSwap',
+      owner: '0xOwnerAddress',
+      tokenIn: USDC_ADDRESS,
+      tokenOut: WETH_ADDRESS,
+      tokenInAmount: 1000000,
+      fee: 0.5,
```

Comment:
test description says "with default values" but seems like this should be `"should create correct market swap action with provided twapFeePool"`

Reply:
ah good catch. i changed the test in between